contracts/eosio.system/src/wps.cpp

```diff
@@ -534,6 +534,40 @@ namespace eosiosystem {
         _proposals.erase(itr_proposal);
     }
 
+    void system_contract::cleanvotes(name reviewer, name proposer, uint64_t begin, uint64_t end){
```

Comment:
We should have a a test case for this function to prove we can clean all votes out for a proposer, using multiple calls, like (begin, end) = (0,1) followed by (1,2) given that there were 2 voters